test/namenode/NameNodeTest.cc

```diff
@@ -0,0 +1,110 @@
+#include "zkwrapper.h"
+#include "zk_nn_client.h"
+
+#include <gtest/gtest.h>
+
+#include <cstring>
+#include <easylogging++.h>
+
+INITIALIZE_EASYLOGGINGPP
+
+namespace {
+
+	class NamenodeTest : public ::testing::Test {
+
+	protected:
+		virtual void SetUp() {
+			system("sudo ~/zookeeper/bin/zkServer.sh stop");
```

Comment:
Not quite sure why that error was happening, may have been a problem with my local zkserver? Anyways, it is structured like this in the rest of the test files now.